operator/deploy/csv/apicurio-registry.clusterserviceversion.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: Streaming & Messaging
+    certified: "false"
+    # DO NOT COMMIT AUTOMATIC MODIFICATIONS TO THIS FILE
+    containerImage: quay.io/apicurio/apicurio-registry-operator:latest-snapshot
```

Comment:
`latest-snapshot` should become `latest`, probably, @carlesarnal to confirm

Reply:
We're using the following tags at the moment:
- (version)
- `latest-release`
- `latest-snapshot`

I think we can have a team discussion about this, but let's deal with it later.

Reply:
ok, we should have this conversation.